crates/solidity/outputs/cargo/tests/src/bindings_output/graph/graphviz.rs

```diff
@@ -63,6 +64,36 @@ impl<'a> DotSubGraph<'a> {
             format!("{graph_id}N{index}", graph_id = self.graph_id,)
         }
     }
+
+    fn node_label(&self, node: GraphNodeRef, node_type: Option<&str>) -> String {
+        let graph_node = &self.graph[node];
+        let mut node_label = if let Some(symbol) = graph_node.attributes.get("symbol") {
+            format!("{symbol} [{index}]", index = node.index())
+        } else if let Some(variable) = graph_node.attributes.get(VARIABLE_DEBUG_ATTR) {
+            format!("{variable} [{index}]", index = node.index())
+        } else {
+            format!("{}", node.index())
```

Comment:
nit: angle brackets?

```suggestion
            format!("[{index}]", index = node.index())
```